console/src/views/Home.module.less

```diff
@@ -29,7 +29,7 @@
 		}
  		.header {
 			.title {
-				margin-left: (4 * @agate-divider-spacing);
+				margin-left: (4 * @agate-heading-spacing-medium);
```

Comment:
Need to verify that `@agate-divider-spacing` and `@agate-heading-spacing-medium` match in size so this margin doesn't get crazy.